r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -974,3 +974,173 @@ test_that("date() errors with unsupported inputs", {
     regexp = "Unsupported cast from double to date32 using function cast_date32"
   )
 })
+
+test_that("difftime works correctly", {
+  test_df <- tibble(
+    time1 = as.POSIXct(
+      c("2021-02-20", "2021-07-31", "2021-10-30", "2021-01-31")
+    ),
```

Comment:
Having pure dates as one of the two time columns we use for general checking is a bit confusing — maybe we should add a comment that these can also be times (or include a few that have times to show that we can mix and match in those argument positions)

Reply:
Done. Some of them are now times.